plugin/trino-hive/src/test/java/io/trino/plugin/hive/TestHiveConnectorTest.java

```diff
@@ -336,6 +339,59 @@ public void testRequiredPartitionFilterInferred()
         assertUpdate(session, "DROP TABLE test_partition_filter_inferred_right");
     }
 
+    @DataProvider
+    private Object[][] queryPartitionFilterRequiredSchemasDataProvider()
+    {
+        return new Object[][]{
+                {"[]"},
+                {"[\"tpch\"]"}
+        };
+    }
```

Comment:
extract data provider extraction to method to separate comit

Reply:
But the data provider is for the new session property that we introduced.